app/download/page.tsx

```diff
@@ -30,84 +31,129 @@ export default function Download() {
+              ) : (
+                // localFormDataあり
+                <th className="px-6 py-4 font-medium text-gray-900">
+                  {formdata.name}
+                </th>
+              )}
+              {/* localFormDataなし */}
+              {formdata === undefined ? (
+                <td className="px-6 py-4 font-medium text-gray-900">
+                  localFormDataなし
+                </td>
+              ) : (
+                // localFormDataあり
+                <td className="px-6 py-4">{formdata.company}</td>
+              )}
+              {/* localFormDataなし */}
+              {formdata === undefined ? (
+                <td className="px-6 py-4">localFormDataなし</td>
+              ) : (
+                // localFormDataあり
+                <td className="px-6 py-4">{formdata.employeeId}</td>
+              )}
+              {/* localFormDataなし */}
+              {formdata === undefined ? (
+                <td className="px-6 py-4">localFormDataなし</td>
+              ) : (
+                // localFormDataあり
+                <td className="px-6 py-4">{formdata.phone}</td>
+              )}
+              {/* localFormDataなし */}
+              {formdata === undefined ? (
+                <td className="px-6 py-4">localFormDataなし</td>
+              ) : (
+                // localFormDataあり
+                <td className="px-6 py-4">{formdata.mail}</td>
+              )}
+
+              <td className="flex justify-end gap-4 px-6 py-4 font-medium">
+                <button type="button" className="text-primary-700">
+                  clearLocalstrage
+                </button>
+              </td>
+            </tr>
+            <tr className="hover:bg-gray-50">
+              <th className="px-6 py-4">
+                <div className="flex h-5 items-center">
+                  <input
+                    type="checkbox"
+                    id="example12"
+                    name="checkGroup1"
+                    className="h-4 w-4 rounded border-gray-300 text-primary-600 shadow-sm focus:border-primary-300 focus:ring focus:ring-primary-200 focus:ring-opacity-50 focus:ring-offset-0 disabled:cursor-not-allowed disabled:text-gray-400"
+                  />
+                </div>
               </th>
-            )}
-            {/* localFormDataなし */}
-            {formdata === undefined ? (
               <td className="px-6 py-4 font-medium text-gray-900">
-                localFormDataなし
+                Helen Howard
+              </td>
+              <td className="px-6 py-4">samplecompany</td>
+              <td className="px-6 py-4">999999</td>
+              <td className="px-6 py-4">999-999-888</td>
+              <td className="px-6 py-4">samplemail@samplemail.com</td>
+              <td className="flex justify-end gap-4 px-6 py-4 font-medium">
+                <a href="hoge">Delete</a>
+                <a href="hoge" className="text-primary-700">
+                  Edit
+                </a>
```

Comment:
This code is old.
Rebase your branch first.
Then request PR.

Reply:
Rebased <button>tags and PR.